elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -44,7 +44,7 @@ class PfeAccordion extends PFElement {
   }
 
   static get observedAttributes() {
-    return ["pfe-disclosure"]
+    return ["pfe-disclosure", "on"]
```

Comment:
Is this still needed after the latest updates?

Reply:
I fixed that too. It's not needed anymore.